src/tz/system/mod.rs

```diff
@@ -175,7 +175,8 @@ fn get_env_tz(db: &TimeZoneDatabase) -> Result<Option<TimeZone>, Error> {
 
     let Some(tzenv) = std::env::var_os("TZ") else { return Ok(None) };
     if tzenv.is_empty() {
-        return Ok(None);
+        // It is commonly agreed (but not standard) that setting an empty `TZ=` uses UTC.
```

Comment:
```suggestion
        // It is commonly agreed (across GNU and BSD tooling at least),
        // but not standard, that setting an empty `TZ=` is indistinguishable
        // from `TZ=UTC`.
```

Slight re-phrase and wrap to 79 columns (inclusive).